.woodpecker/.test-pr.yml

```diff
@@ -1,31 +1,31 @@
 steps:
   install:
-    image: danlynn/ember-cli:5.8.0-node_20.12
+    image: danlynn/ember-cli:5.12.0-node_22.9
     commands:
       - npm ci
   lint-js:
-    image: danlynn/ember-cli:5.8.0-node_20.12
+    image: danlynn/ember-cli:5.12.0-node_22.9
     group: lint
```

Comment:
The `group` part is no longer supported, so I would remove it (the individual steps are fast, so no real speed increase by running them in paralel). 

Or you can do something like this: https://github.com/lblod/frontend-dashboard/pull/48/files#diff-3a923165ee9ff553661dcec110075b93a6024990025b51285f62586c6c6d3495